src/commonMain/kotlin/org/cru/godtools/tool/xml/XmlPullParserFactory.kt

```diff
@@ -0,0 +1,5 @@
+package org.cru.godtools.tool.xml
+
+abstract class XmlPullParserFactory {
```

Comment:
Is this code scanning check anything to consider?

Reply:
I'll probably follow it eventually, was waiting to see how these APIs evolve as I implement stuff and integrate it into the actual apps